hardware/simulation/icarus/Makefile

```diff
@@ -4,8 +4,6 @@ CACHE_DIR:=../../..
 incdir:=-I
```

Comment:
remove  comment above "#module paths"

Reply:
The comment #module paths was in the original makefile.
There are others.
Should all makefiles be comments free?